manifests/c/Creosys/OBDAutoDoctor/3.8.2/Creosys.OBDAutoDoctor.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Creosys.OBDAutoDoctor
+PackageVersion: 3.8.2
+PackageLocale: en-US
+Publisher: Creosys
+PackageName: OBD Auto Doctor
+License: Copyright 2010-2021 Creosys Ltd.
+ShortDescription: OBD Auto Doctor Setup

```

Comment:
```
ShortDescription: OBD Auto Doctor Setup
```
Could you please write a better description 👍

Reply:
Thank you for your suggestion. I have updated the ShortDescription.